week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,23 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+   console.log("The ball has shaken!");
```

Comment:
nice try, Hacer.
now when you know arrays and if/else would like to think about how you could implement some random element into this function?

there are 5 positive options, 5 very positive, also bunch of other options, 4 x 5 total.
But we always use here only one: " Outlook good " --> with extra spaced around, and we always git it "positive" (with small 1st letter) value. But what about all the others? 
Ask a question, if you need any tip/hint/help :)